src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationServiceImpl.java

```diff
@@ -66,6 +79,46 @@ public Integer process(MultipartFile document, Annotation annotation) {
         return annotationEntity.getId();
     }
 
+    @Override
+    public AnnotationResponseDto downloadAnnotationDoc(Integer annotationId, Integer annotationDocumentId) {
+
+        final Optional<ExternalObjectDirectoryEntity> dirEntity = eodRepository.findAnnotationIdAndAnnotationDocumentId(annotationId, annotationDocumentId);
```

Comment:
`dirEntity` is not very descriptive, should probably be something like `annotationDocEodEntityOpt`